### DISCUSSION TITLE
网站页面"使用教程-基本用法"报错

### DISCUSSION THREAD

**Discussion by 200cc** [jd-opensource/drip-table#53] [General]
http://drip-table.jd.com/drip-table-generator/functions/  页面报错: TypeError: Cannot read properties of undefined (reading '0')

**Comment by helloqian12138** [comment]
因为页面文档的示例代码没有更新，使用错误，导致传值失败。我这边马上修复发一版。

**Comment by helloqian12138** [comment]
已修复
